root/static/scripts/common/components/ReleaseEvents.js

```diff
@@ -0,0 +1,156 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React, {useCallback, useState} from 'react';
+
+import CountryAbbr from '../../../../components/CountryAbbr';
+import hydrate from '../../../../utility/hydrate';
+import {bracketedText} from '../utility/bracketed';
+import formatDate from '../utility/formatDate';
+import isDateEmpty from '../utility/isDateEmpty';
+
+import EntityLink from './EntityLink';
+
+const COLLAPSE_THRESHOLD = 3;
```

Comment:
This follows the other comment on hiding at least two elements. If you still want to show at least the 2 first elements:
```suggestion
const COLLAPSE_THRESHOLD = 4;
```